jasper_reports/JasperReports/browse_data_generator.py

```diff
@@ -312,14 +312,14 @@ class CsvBrowseDataGenerator(BrowseDataGenerator):
     # they will imply a LEFT JOIN like behaviour on the rows to be shown.
     def generate(self, file_name):
         self.all_records = []
-        relations = self.report.relations()
+        relations = self.report.relations
 
         # The following loop generates one entry to allRecords list
         # for each record that will be created. If there are any relations
         # it acts like a LEFT JOIN against the main model/table.
-        reportCopies = self.report.copies() or 1
+        reportCopies = self.report.copies or 1
         sequence = 0
-        copiesField = self.report.copiesField()
+        copiesField = self.report.copies_field
         for record in self.env[self.model].browse(self.ids):
```

Comment:
The method needs to be copiesField, any reason you are making it different? Did you change the def?